Tests/Fakes/FakeSession.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+class FakeSession: NSURLSession {
+  var providedRequest: NSURLRequest?
+  var dataTask: FakeDataTask?
+  var completionHandler: ((NSData?, NSURLResponse, NSError?) -> Void)?
```

Comment:
Good catch. I don't remember why I originally added this, but it must have been a dead end.